android/ui/src/main/java/com/trafi/ui/theme/Color.kt

```diff
@@ -136,3 +136,6 @@ class MaasColorPalette(
         isLight = other.isLight
     }
 }
+
+fun String.parseColor(): Color =
+    Color(android.graphics.Color.parseColor("#$this"))
```

Comment:
We'd discussed with @TomasSkar that we want to eventually move this to Kotlin/Native so that iOS could use the same function, too 🙂

Reply:
https://github.com/trafi/maas-components/pull/64